### DISCUSSION TITLE
Polkadot support

### DISCUSSION THREAD

**Discussion by Douglasacost** [WalletConnect/walletconnect-monorepo#1791] [Developer Support] [answered]
Is there any plan for support Polkadot/Substrate based wallets?

**Comment by chadyj** [comment, marked as answer]
Yes, Polkadot is supported in v2.

**Reply by Douglasacost** [reply]
@chadyj Thanks for the answer, could you refer me to the correct documentation where is an example for implement wallet connect with polkadot?

**Reply by chadyj** [reply]
You can find it in our sample apps

https://react-app.walletconnect.com/
https://react-wallet.walletconnect.com/

Grab the source here https://github.com/WalletConnect/web-examples
